weave/trace_server/calls_query_builder.py

```diff
@@ -784,7 +786,7 @@ def _as_sql_base_format(
         {feedback_where_sql}
         {id_mask_sql}
         {id_subquery_sql}
-        {call_parts_predicate_filters_sql}
+        {str_filter_opt_sql}
```

Comment:
@chance-wnb just modified this code path. Please update with master before merging

Reply:
I think those changes should be in this version, just merged again to be safe (and got that print statement)

Reply:
👍